vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -601,9 +602,15 @@ void ViewViewManager::TryUpdateView(
     pViewShadowNode->GetControl().Content(visualRoot);
 
   if (useControl && pViewShadowNode->IsAccessible() != pViewShadowNode->IsFocusable()) {
```

Comment:
```suggestion
  if (useControl && (pViewShadowNode->IsAccessible() != pViewShadowNode->IsFocusable())) {
```

Reply:
Maybe let's add a set of parentheses here for some clarity/will ensure proper order of operations.

Reply:
Not sure that's needed here because the second expression is a comparison and not a & and | operator